core/src/main/java/binnie/core/liquid/AlcoholEffect.java

```diff
@@ -18,6 +18,8 @@ public static void makeDrunk(final EntityPlayer player, final float strength) {
 		if (slowIntense < 0.0f) {
 			slowIntense = 0.0f;
 		}
+
+		// TODO blindIntense
```

Comment:
What is the TODO for?

Reply:
The variable should be used for the effect of blindness, but the code is not complete.